packages/react-core/src/components/Alert/examples/Alert.md

```diff
@@ -147,14 +303,13 @@ import { Alert } from '@patternfly/react-core';
 </React.Fragment>
 ```
 
-### Inline plain variations
+#### Plain variations
 
-It is not recommended to use an inline plain alert with actionClose nor actionLinks.
+It is not recommended to use an inline plain alert with `actionClose` nor `actionLinks`.
```

Comment:
Unless it could fit in either the design guidelines or a11y tabs, including why it's not recommended may be a good addition, but I wouldn't block over it.